.github/workflows/update-demisto-sdk-version.yml

```diff
@@ -1,20 +1,32 @@
 name: update-demisto-sdk-version
-on: workflow_dispatch
+on:
+  workflow_dispatch:
+    inputs:
+      reviewer:
+        description: The reviewer of the PR.
+        required: true
+        type: string
+      release_version:
+        description: The SDK release version
+        required: true
+        type: string
+
+permissions:
+  contents: write
 
 jobs:
   update-dep:
     runs-on: ubuntu-latest
-    permissions:
-      pull-requests: write
 
     steps:
     - name: Install poetry
       uses: Gr1N/setup-poetry@v8
 
     - name: Checkout
-      uses: actions/checkout@v3
+      uses: actions/checkout@v2
```

Comment:
why not v3?

Reply:
@dorschw 
because of [this](https://jonathansoma.com/everything/git/github-actions-403-error/#:~:text=First%2C%20make%20sure%20you%27re%20using%20actions/checkout%402%20in%20your%20steps%20section.%20It%27s%20older%20than%20actions/checkout%40v3%2C%20but%20it%27s%20the%20one%20that%20works%20for%20me.)...